libr/core/cmd_print.c

```diff
@@ -2288,21 +2288,19 @@ static int cmd_print(void *data, const char *input) {
 		case 'j':
 			{
 				char *str, *type;
-				ut64 vaddr;
-				RIOSection *section;
+				RBinSection *section;
 				if (input[2] == ' ' && input[3]){
 					len = r_num_math (core->num, input+3);
 					len = R_MIN (len, core->blocksize);
 				}
-				vaddr = r_io_section_offset_to_vaddr (core->io, core->offset);
-				section = core->io->section;
-				if (!section)
-					vaddr = UT64_MAX;
+				// XXX: we should cross-check with r_io to see if it's really mapped
+				section = r_bin_get_section_at (r_bin_cur_object (core->bin),
```

Comment:
We should get the section from RIO, but since so far we have always mapped all sections, this should be exactly the same of what we had.